modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/PlaylistDao.kt

```diff
@@ -98,6 +105,21 @@ abstract class PlaylistDao {
         }
     }
 
+    @Query(
+        """
+        SELECT
+          COUNT(*) AS episode_count,
+          SUM(MAX(0, IFNULL(podcastEpisode.duration, 0) - IFNULL(podcastEpisode.duration, 0))) AS time_left
+        FROM playlists AS playlist
+        JOIN manual_playlist_episodes AS playlistEpisode ON playlistEpisode.playlist_uuid IS playlist.uuid
```

Comment:
```suggestion
        JOIN manual_playlist_episodes AS playlistEpisode ON playlistEpisode.playlist_uuid = playlist.uuid
```
Should this just be an equals symbol?

Reply:
Why?

Reply:
Again, why not?

Reply:
This is in the abstract because UUIDs can't be null. But if an episode is defined with a null UUID, why wouldn’t we want to include it? If a null UUID is considered valid, I don’t see a reason to skip it. If it’s not a valid UUID, then it shouldn’t be allowed in the database in the first place. Which is the case in our situation.